docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -135,6 +135,7 @@ Limitations
 
 The following SQL statements aren't  supported:
 
+* :doc:`/sql/delete`
```

Comment:
I don't see this limitation mentioned for druid

Reply:
Druid currently has nothing documented about supported/unsupported SQL statements. While we could add that DELETE is unsupported in Druid, we would also need to add info on what other SQL statements are supported or it'll appear that DELETE is the only limitation.

I'd propose we track that work separately as it's beyond scope for this issue.

Reply:
We will shorty follow up with adding SQL support sections for all these connectors. The current need is to update the docs with regards to the DELETE statement.

Reply:
```
.. include:: sql-delete-limitation.fragment
```

Reply:
The fragment shouldn't be necessary here, since it only applies to JDBC connectors that do support DELETE and this one doesn't.